source/basic.tex

```diff
@@ -6169,9 +6169,13 @@
 \end{note}
 
 \pnum
-Two expression evaluations \defn{conflict} if one of them modifies a memory
+Two expression evaluations \defn{conflict} if one of them modifies\iref{defns.access} a memory
 location\iref{intro.memory} and the other one reads or modifies the same
 memory location.
+\begin{note}
+When a modification alters no bits,
+this does not prevent if from conflicting.
```

Comment:
```suggestion
this does not prevent it from conflicting.
```

Reply:
How about:

> A modification can still conflict even if it does not alter the value of any bits.

I'm a little unhappy about just saying "can conflict" rather than "can conflict with another expression evaluation" but that starts to get too wordy again.

An even wordier form of the same sentence would be:

> A modification can still conflict with another expression evaluation even if it does not alter the value of any bits in the memory location.

Reply:
I think
> A modification can still conflict even if it does not alter the value of any bits.

is the best option out of these. Since "to conflict" has only been defined for expression evaluations, it is immediately obvious what the modification can conflict with, so we don't need to be so explicit.